### PR TITLE
add serialization for `ignore_merges`

### PR DESCRIPTION
**Pull request by ArthurZucker**
Make sure we serialize `ignore_merges`

### CODE REVIEW

**Review by pcuenca — APPROVED** [review]
LGTM!

File: tokenizers/src/models/bpe/serialization.rs [hunk 1]
```diff
@@ -136,3 +143,45 @@ impl<'de> Visitor<'de> for BPEVisitor {
...
+            .cloned()
+            .collect();
+        let bpe = BpeBuilder::default()
+            .vocab_and_merges(vocab, vec![])
+            .unk_token("<unk>".to_string())
+            .ignore_merges(true)
+            .build()
+            .unwrap();
+
+        let bpe_string = r#"{"type":"BPE","dropout":null,"unk_token":"<unk>","continuing_subword_prefix":null,"end_of_word_suffix":null,"fuse_unk":false,"byte_fallback":false,"ignore_merges":true,"vocab":{"<unk>":0,"a":1,"b":2},"merges":[]}"#;
```
Comment (Narsil) [inline tokenizers/src/models/bpe/serialization.rs:184]: Can you try deserializing when the field is missing too ?

Comment (ArthurZucker) [reply tokenizers/src/models/bpe/serialization.rs:184]: yep!

**Review by Narsil — APPROVED** [review]
LGTM

**Pull request merged**